x/staging/posts/types/msgs.go

```diff
@@ -333,12 +329,12 @@ func (msg MsgAnswerPoll) ValidateBasic() error {
 	}
 
 	if len(msg.Answers) == 0 {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "provided answer must contains at least one answer")
+		return sdkerrors.Wrap(ErrPollInvalidAnswers, "provided answers must contains at least one answer")
 	}
 
 	for _, answer := range msg.Answers {
 		if strings.TrimSpace(answer) == "" {
-			return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "invalid answer")
+			return ErrPollEmptyAnswer
```

Comment:
I think it's better to use a more generic `ErrInvalidPollAnswer` and specify the error here